plugins/catalog-react/api-report.md

```diff
@@ -157,6 +161,32 @@ export type EntityFilter = {
   toQueryValue?: () => string | string[];
 };
 
+// @public
+export class EntityGenericFilter implements EntityFilter {
```

Comment:
Didn't quite get to discuss the naming in the original issue much, but I think it needs a bit of work. What do you think about something like `EntityFieldFilter` instead?